Tests/AutoCollection/Statsbeat.tests.ts

```diff
@@ -275,8 +275,8 @@ describe("AutoCollection/Statsbeat", () => {
         it("Multiple network categories and endpoints", (done) => {
             statsBeat.enable(true);
             const sendStub = sandbox.stub(statsBeat, "_sendStatsbeats");
-            statsBeat.countRequest(0, "breezeFirstEndpoint", 100, true);
-            statsBeat.countRequest(1, "quickpulseEndpoint", 200, true);
+            statsBeat.countRequest(0, "https://breezeFirstEndpoint.something.com", 100, true);
+            statsBeat.countRequest(1, "http://quickpulseEndpoint.something.com", 200, true);
```

Comment:
add a test with www. prefix that way it highlights the issue above with the extra `\` after the `www` in the regex